### DISCUSSION TITLE
Using global layers within a Slidev theme

### DISCUSSION THREAD

**Discussion by openscript** [slidevjs/slidev#1130] [Q&A]
I've the following questions regarding global layers and Slidev themes:

1. Should global layers be used in Slidev themes?
2. Can global layers from a Slidev theme be overwritten in a presentation using that theme?

**Comment by damienalexandre** [comment]
Yes and yes.

Just has to read the code and discovered how it works.

- your theme can indeed add Global Layers, you just have to put the `.vue` files at the root of your theme
- if your presentation also has the same `.vue` file at it's root, it overwrite the theme one
